### PR TITLE
covid banner updated

### PR DESCRIPTION
**Pull request by Muhammad-Ali-Ahmad**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -260,6 +260,7 @@ label.help_improve_gov_uk=Helpwch i wella gwasanaethau CThEM
 label.get.involved.in.making.gov.services.better=Gallwch fod yn rhan o wneud gwasanaethau’r Llywodraeth yn well (yn agor ffenestr neu dab newydd)
 label.no_thanks=Dim diolch
 label.url_coronavirus=Coronafeirws (COVID-19): yr hyn y mae angen i chi ei wneud (yn agor ffenestr neu dab newydd)
```
Comment (oc-ste) [inline conf/messages.cy:262]: Remove this as its overridden below

File: app/assets/stylesheets/pertaxMain.scss [hunk 2]
```diff
@@ -156,14 +156,29 @@ input[type="text"].autocomplete__input {
...
+
+    display: inline-block;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-align: center;
+    align-items: center;
+    box-sizing: border-box;
+    padding-left: 50px;
+    min-height: 45px;
+    background-image: url(https://assets.publishing.service.gov.uk/static/covid-arrow-af5ae4c3e1f65811d4e472b4a5dace2950508ad3ef956b54e0cdfc80aef96419.png);
```
Comment (oc-ste) [inline app/assets/stylesheets/pertaxMain.scss:168]: should we be hosting the image or is this standard asset stuff?

File: conf/messages.cy (hunk 1 above)
Comment (Muhammad-Ali-Ahmad) [reply conf/messages.cy:262]: i kept the original so when we do get translation, we only have to replace "guidance and support" bit and dont have to look for welsh for "opens in a new window or tab"

File: app/assets/stylesheets/pertaxMain.scss (hunk 2 above)
Comment (Muhammad-Ali-Ahmad) [reply app/assets/stylesheets/pertaxMain.scss:168]: thats where gov.uk is hosting the image

**Pull request merged**